### PR TITLE
[Documentation] Updated from OpenMCTWeb to OpenMCT

### PR DESCRIPTION
**Pull request by ZachMoreno**
minor change & please let me know if I can help with anything else

# Author Checklist:

Changes appear to address issue? Y
Appropriate unit tests included? N/A - No code changes
Code style and in-line documentation are appropriate? N/A
Commit messages meet standards? N

EDIT: [CLA](https://nasa.github.io/openmct/contributing/ind-cla-open-mct.pdf) has been completed, signed and sent to appropriate email address and cc-ed to arc-dl-openmct@mail.nasa.gov.

### CODE REVIEW

File: docs/src/tutorials/index.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ To run the tutorial web server
 
 Once running, you should be able to view Open MCT from your browser at 
 http://localhost:8080/ (assuming the web server is running on port 8080, 
-and OpenMCTWeb is installed at the server's root path). 
+and OpenMCT is installed at the server's root path).
```
Comment (akhenry) [inline docs/src/tutorials/index.md]: Open MCT

File: docs/src/tutorials/index.md [hunk 2]
```diff
@@ -79,7 +79,7 @@ changes to stylesheets, or you are running the minified version of the app
 
 The next step is to run a web server so that you can view the Open MCT 
 client (including the plugins you add to it) in browser. Any web server can 
-be used for hosting OpenMCTWeb, and a trivial web server is provided in this 
+be used for hosting OpenMCT, and a trivial web server is provided in this
```
Comment (akhenry) [inline docs/src/tutorials/index.md]: Open MCT

**Comment by akhenry** [comment]
@ZachMoreno Nice! Thanks for doing this. Also I've got your CLA, thanks for sending it along. 

Just one thing, we have standardised on "Open MCT" with a space, so could you do us a favor and fix that as well while you're at it? Just in the two lines that I've commented on.

Thanks!

**Comment by akhenry** [comment]
@ZachMoreno Well spotted! I've got your CLA as well, thanks for sending that.

Just one thing. We have standardised on "Open MCT" - with a space. Would you mind replacing OpenMCT with Open MCT in the two spots that I commented on while you're at it? Thanks!

**Comment by ZachMoreno** [comment]
@akhenry Thanks for catching that & bringing me up to speed on the standard. The adjustment you requested has been completed.

Thanks Kindly!

**Comment by akhenry** [comment]
@ZachMoreno No problem, thanks for fixing our inconsistent documentation!

### Reviewer Checklist

1. Changes appear to address issue? Y
2. Appropriate unit tests included? N/A no code changes
3. Code style and in-line documentation are appropriate? N/A no code changes
4. Commit messages meet standards? Y

**Comment by ZachMoreno** [comment]
Thanks Kindly!

**Pull request merged**